src/Native/System.Device.Gpio.Native/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+project (System.Device.Gpio.Native C)
+
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+add_library(System.Device.Gpio.Native
+	SHARED
```

Comment:
NIT: These two lines have tabs, can we indent with spaces instead?

Reply:
sure